src/entities/Proposal/templates/messages.ts

```diff
@@ -2,10 +2,15 @@ import { Vote } from '../../Votes/types'
 import { calculateResult } from '../../Votes/utils'
 import { ProposalAttributes, ProposalStatus } from '../types'
 
+function getProposalStatusDisplayName(proposalStatus: ProposalStatus) {
+  if (proposalStatus === ProposalStatus.OutOfBudget) return 'OUT OF BUDGET'
```

Comment:
You can do something like `ProposalStatus.OutOfBudget.split('_').join(' ').toUpperCase()`